### ISSUE TITLE
[Bug] Building OpenVINO samples after build fails

### ISSUE THREAD

**Issue by marcjasner** [openvinotoolkit/openvino#13755]
##### System information (version)
- OpenVINO=> Latest pulled from git on 10/31/2022
- Operating System / Platform => Ubuntu 20.04.4 LTS
- Compiler => gcc 9.4.0
- Problem classification => Build Issue

##### Detailed description
After building and installing OpenVINO (pulled from git on 10/31/2022 and built using the instructions on https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux) I moved to the installation instructions at https://github.com/openvinotoolkit/openvino/wiki/BuildingCode#installing. While building the samples I get the following output:

```
:~/openvino_103122_github/samples/cpp$ ./build_samples.sh

Setting environment variables for building samples...
[setupvars.sh] OpenVINO environment initialized
-- The C compiler identification is GNU 9.4.0
-- The CXX compiler identification is GNU 9.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- gflags (2.2.2) is found at /usr/lib/x86_64-linux-gnu/cmake/gflags
-- /usr/bin/pkg-config: zlib (1.2.11) is found at /usr
-- nlohmann_json (3.7.3) is found at /usr/lib/cmake/nlohmann_json
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
CMake Warning at benchmark_app/CMakeLists.txt:27 (MESSAGE):
  OpenCL is disabled or not found, benchmark_app will be built without OpenCL
  support.  Install OpenCL.


CMake Warning at benchmark_app/CMakeLists.txt:56 (MESSAGE):
  OpenCL CPP header is not found, benchmark_app will be built without OpenCL
  support.  Download it from: https://github.com/KhronosGroup/OpenCL-CLHPP
  and set -Dopencl_root_hints=[PATH]/OpenCL-CLHPP/include to cmake.


-- Configuring done
-- Generating done
-- Build files have been written to: /home/marc/openvino_cpp_samples_build
Consolidate compiler generated dependencies of target cnpy
Consolidate compiler generated dependencies of target ie_samples_utils
[  6%] Built target cnpy
[ 18%] Built target ie_samples_utils
Consolidate compiler generated dependencies of target hello_query_device
Consolidate compiler generated dependencies of target speech_sample
Consolidate compiler generated dependencies of target format_reader
[ 24%] Built target hello_query_device
[ 33%] Built target speech_sample
[ 51%] Built target format_reader
Consolidate compiler generated dependencies of target classification_sample_async
Consolidate compiler generated dependencies of target hello_classification
Consolidate compiler generated dependencies of target hello_nv12_input_classification
Consolidate compiler generated dependencies of target benchmark_app
[ 57%] Built target hello_classification
Consolidate compiler generated dependencies of target model_creation_sample
[ 69%] Built target classification_sample_async
[ 69%] Built target hello_nv12_input_classification
[ 75%] Building CXX object hello_reshape_ssd/CMakeFiles/hello_reshape_ssd.dir/main.cpp.o
[ 75%] Building CXX object benchmark_app/CMakeFiles/benchmark_app.dir/main.cpp.o
[ 81%] Built target model_creation_sample
/home/marc/openvino_103122_github/samples/cpp/hello_reshape_ssd/main.cpp:11:10: fatal error: openvino/opsets/opset9.hpp: No such file or directory
   11 | #include "openvino/opsets/opset9.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [hello_reshape_ssd/CMakeFiles/hello_reshape_ssd.dir/build.make:76: hello_reshape_ssd/CMakeFiles/hello_reshape_ssd.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:524: hello_reshape_ssd/CMakeFiles/hello_reshape_ssd.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
/home/marc/openvino_103122_github/samples/cpp/benchmark_app/main.cpp: In function ‘ov::hint::PerformanceMode get_performance_hint(const string&, const ov::Core&)’:
/home/marc/openvino_103122_github/samples/cpp/benchmark_app/main.cpp:170:59: error: ‘CUMULATIVE_THROUGHPUT’ is not a member of ‘ov::hint::PerformanceMode’
  170 |                 ov_perf_hint = ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT;
      |                                                           ^~~~~~~~~~~~~~~~~~~~~
/home/marc/openvino_103122_github/samples/cpp/benchmark_app/main.cpp: In function ‘int main(int, char**)’:
/home/marc/openvino_103122_github/samples/cpp/benchmark_app/main.cpp:1231:21: error: ‘serialize’ is not a member of ‘ov’
 1231 |                 ov::serialize(compiledModel.get_runtime_model(), FLAGS_exec_graph_path);
      |                     ^~~~~~~~~
make[2]: *** [benchmark_app/CMakeFiles/benchmark_app.dir/build.make:90: benchmark_app/CMakeFiles/benchmark_app.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:357: benchmark_app/CMakeFiles/benchmark_app.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
Error on or near line 92; exiting with status 1

```

##### Steps to reproduce

Pull git from OpenVINO repo.
Configure with the following CMake command:
`
cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_PYTHON=ON -DPYTHON_EXECUTABLE=`which python3.8` -DPYTHON_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.8.so -DPYTHON_INCLUDE_DIR=/usr/include/python3.8 -DENABLE_INTEL_CPU=ON -DENABLE_INTEL_GPU=ON -DENABLE_OV_ONNX_FRONTEND=ON ..`

Build OpenVINO.
Install with:
` cmake --install . --prefix ~/openvino_103122_github/`

Go to openvino_103122_github folder. Run setupvars.sh script.
Go to samples/cpp directory. Run build_samples.sh script.

Error will occur

##### Issue submission checklist

 - [X] I report the issue, it's not a question
   
 - [X] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [X] There is reproducer code and related data files: images, videos, models, etc.

**Comment by ilya-lavrenov** [comment]
Hi @marcjasner 
Have not you installed older OpenVINO version to your system? Please, check that there is no OpenVINO headers in `/usr/include` and libs in `/usr/lib/x86_64-linux-gnu/`

**Comment by marcjasner** [comment]
@ilya-lavrenov,

Yes, I installed an older version and am trying to build the latest version to keep up to date.

I do not see any openvino folders/headers in those locations.

**Comment by ilya-lavrenov** [comment]
Looks like older version of OpenVINO is used with latest version of samples. There is a mismatch because samples demonstrate usage of newer API.
Could you please provide more info?
- where OpenVINO is installed
- go to build folder with samples and find value of OpenVINO_DIR in CMakeCache.txt file

**Comment by marcjasner** [comment]
OpenVINO 2022.1 was previous installed in ~/intel/openvino which is a symlink to ~/intel/openvino_2022.1.0.643

Do you mean the build folder for the build of OpenVINO that I just completed from github? In that case:
`//The directory containing a CMake configuration file for OpenVINO.
OpenVINO_DIR:PATH=/home/marc/src/openvino_github_103122/openvino/build
`
That is the directory where I had just built the github checkout I made last night

**Comment by ilya-lavrenov** [comment]
Are you running this command from source tree?
```
:~/openvino_103122_github/samples/cpp$ ./build_samples.sh
```
I suppose once you build OV, you need to install it to some directory (`cmake -DCMAKE_INSTALL_PREFIX=some_dir -P cmake_install.cmake` and run samples build using `some_dir/samples/cpp/build_samples.sh`

**Comment by marcjasner** [comment]
According to the wiki the samples build should be done in the install directory after running:

cmake --install <BUILDDIR> --prefix <INSTALLDIR>

It says to then do the following:

cd <INSTALLDIR>/samples/cpp
./build_samples.sh

________________________________

**Comment by marcjasner** [comment]
I think I fixed it. It seems the OpenVINO_DIR variable was not getting updated and kept pointing to the old version of OpenVINO. I removed the setupvars.sh from the previous install of OpenVINO from my .bashrc, logged out and logged back in, and now the samples build.

The only problem I have now is that when building OpenVINO the wheel won't build. I get the following:

`-- Python version=python3.8
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'wheel.vendored'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'wheel.vendored'
CMake Error at src/bindings/python/wheel/CMakeLists.txt:38 (string):
  string sub-command STRIP requires two arguments.


CMake Error at src/bindings/python/wheel/CMakeLists.txt:39 (string):
  string sub-command STRIP requires two arguments.
`

Wheel is installed, though I don't know what wheel.vendored is.
